collect_app/src/test/java/org/odk/collect/android/utilities/TextUtilsTest.java

```diff
@@ -22,17 +22,34 @@ public void textToHtml_BouncesNullInput() {
         Assert.assertNull(observed);
     }
 
+    @Test
+    public void markDownToHtmlEscapesBackslash() {
+        String[][] tests = {
+                {"A\\_B\\_C", "A_B_C"},
+                {"_A\\_B\\_C_", "<em>A_B_C</em>"},
+                {"A_B\\_C", "A_B_C"},
+                {"A\\_B_C", "A_B_C"},
+                {"\\__AB\\__", "_<em>AB_</em"},
```

Comment:
Looks like a > is missing.

Reply:
oh, my bad. I will make add the missing > to the code

Reply:
I wonder how this test passed without it. Please do double check that it is indeed being run!

Reply:
My thoughts exactly